src/test/java/teammates/test/cases/ui/pagedata/InstructorCourseDetailsPageDataTest.java

```diff
@@ -59,9 +59,9 @@ public void testAll() {
         pageData.init(curInstructor, courseDetails, instructors, students);
         
         assertEquals(instructors.size(), pageData.getInstructors().size());
-        assertEquals(students.size(), pageData.getStudentsTable().getRows().size());
-        assertEquals(4, pageData.getStudentsTable().getRows().get(0).getActions().size());
-        assertEquals(2, pageData.getStudentsTable().getRows().get(0).getCommentActions().size());
+        //assertEquals(students.size(), pageData.getStudentsTable().getRows().size());
```

Comment:
Down by 0.1%. It could very well be because of the now-deleted classes, which previously are covered (and thus adding to the %). Should I just remove all the tests altogether?
The UI template used here has been covered in another page data, the one that first used it.